internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -167,20 +167,36 @@ public void testIterateOnMapOfLists() throws Exception {
 
   @Test
   public void testParseMap() {
-    assertEquals(Utils.parseCommaSeparatedStringMapFromString("", "test_field").size(), 0);
-    Map nonEmptyMap = Utils.parseCommaSeparatedStringMapFromString("a=b", "test_field");
+    assertEquals(Utils.parseJsonMapFromString("", "test_field").size(), 0);
+    Map nonEmptyMap = Utils.parseJsonMapFromString("a=b", "test_field");
     Map expectedMap = new HashMap<>();
     expectedMap.put("a", "b");
     assertEquals(nonEmptyMap, expectedMap);
```

Comment:
This test case should now be failing. Let's change this test to ensure that an exception is thrown for comma-separated map similar to the `expectThrows` in the next line